src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
```

Comment:
Can't understand where are you using this global var. it's here just for fun ? delete code which is not in use anymore